packages/inventory/src/EntityDetail.js

```diff
@@ -46,9 +50,16 @@ class EntityDetails extends Component {
         });
     };
 
+    handleModalToggle = () => {
+        this.setState({
+            isModalOpen: !this.state.isModalOpen
+        });
+    };
+
     generateTop = () => {
-        const { entity, loaded, actions } = this.props;
+        const { entity, loaded, actions, deleteEntity, addNotification } = this.props;
         const { isOpen } = this.state;
+        const inventoryActions = [{ onClick: this.handleModalToggle, title: 'Delete' }, ... actions || [] ];
```

Comment:
Should not the button be red as in Sources to be consistent?

Reply:
Nope PF now says that destructive actions in dropdowns should have regular color.

Reply:
Ahhh....